recipes/tl-expected/all/conanfile.py

```diff
@@ -37,6 +37,7 @@ def package_id(self):
         self.info.header_only()
 
     def package_info(self):
+        self.cpp_info.name = "tl"
         self.cpp_info.names["cmake_find_package"] = "tl-expected"
         self.cpp_info.names["cmake_find_package_multi"] = "tl-expected"
         self.cpp_info.components["expected"].name = "expected"
```

Comment:
I am not sure the syntax is expressive enough to I can match what it output by this project.